docs/the-new-architecture/pure-cxx-modules.md

```diff
@@ -15,7 +15,7 @@ In this guide, we will go through the creation of a pure C++ Turbo Native Module
 The rest of this guide assume that you have created your application running the command:
 
 ```shell
-npx @react-native-community/cli@latest init SampleApp --version 0.80.0
+npx @react-native-community/cli@latest init SampleApp --version 0.82.0
```

Comment:
Hey @riteshshukla04, nice catch! 👍 

Let's use `import {getCurrentVersion} from '@site/src/getCurrentVersion';` helper here and update also all versioned docs. You can find some usage examples in other docs.